st2common/st2common/util/jinja.py

```diff
@@ -180,3 +187,17 @@ def is_jinja_expression(value):
             return True
 
     return False
+
+
+def convert_jinja_to_raw_block(value):
+    if isinstance(value, dict):
+        return {k: convert_jinja_to_raw_block(v) for k, v in six.iteritems(value)}
+
+    if isinstance(value, list):
+        return [convert_jinja_to_raw_block(v) for v in value]
+
+    if isinstance(value, six.string_types):
+        if JINJA_REGEX_PTRN.findall(value) or JINJA_BLOCK_REGEX_PTRN.findall(value):
```

Comment:
@Kami it doesn't matter if this whole value is in raw to be honest. It's not being used in Mistral. If used at all, it's only for references. I don't expect people to use the action chain parameters from the context. I don't even think they should be in the context passing into Mistral. I don't want to remove it yet as I don't know which poor soul out there is referencing them.